pkgs/development/python-modules/chainer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, pkgs, lib
+, python, buildPythonPackage, fetchPypi
+, numpy, cudaSupport ? false
+}:
+
+with pkgs;
+with python.pkgs;
```

Comment:
Instead of using `python.pkgs`, it is preferred to list the python dependencies as arguments to the derivation.